pkg/crawl/identify.go

```diff
@@ -227,10 +207,23 @@ func (i *Log4jIdentifier) lookForMatchInZip(ctx context.Context, depth uint, r *
 	return archiveResult, versions, nil
 }
 
-// boolean returned is whether the version was matched.
-func lookForMatchInFileInZip(path string, size int64, contents io.Reader) (Finding, string, bool) {
+// checkForObfuscation applies a heuristic to determine if a class appears to have been obfuscated.
```

Comment:
I think it's worth adding the human-readable form of what comparison/determination is being done (the `if 0 < averageSizes.PackageName && averageSizes.PackageName < i.ObfuscatedPackageNameAverageLength && 0 < averageSizes.ClassName && averageSizes.ClassName < i.ObfuscatedClassNameAverageLength` line of logic is dense enough that explaining what it's doing would be helpful)